content/develop/reference/modules/modules-api-ref.md

```diff
@@ -1934,9 +1932,8 @@ Redis will accumulate all the calls to this function in the context of
 the callback, and will propagate all the commands wrapped in a MULTI/EXEC
 transaction. However when calling this function from a threaded safe context
 that can live an undefined amount of time, and can be locked/unlocked in
```

Comment:
Guessing the "in" shouldn't be there?
```suggestion
that can live an undefined amount of time, and can be locked/unlocked
```

Reply:
There's no point in changing text, as it's all generated from the redis/redis source code. That said, I'll make a note of this for when I start spelunking and requesting changes.